go.mod

```diff
@@ -4,12 +4,16 @@ go 1.12
 
 require (
 	github.com/BurntSushi/toml v0.3.1
-	github.com/containers/buildah v1.11.2
+	github.com/containers/buildah v1.11.2 // indirect
```

Comment:
What is pulling in buildah?  This is what we are trying to avoid.

Reply:
buildah is not used in the code. I nede to run go mod tidy to clean.